content/logs/languages/php.md

```diff
@@ -0,0 +1,503 @@
+//use Monolog\Formatter\JsonFormatter;
+  
+// with the monolog instance
+$monolog = ...
+  
+///// Log shipper configuration
+
+$formatter = new JsonFormatter();
+$stream = new StreamHandler(__DIR__.'/application-json.log', Logger::DEBUG);
+$stream->setFormatter($formatter);
+
+$monolog->pushHandler($stream);
+return $r;});
+```
+
+and then [stream your log files to Datadog](/logs/#custom-log-collection)
+
+#### Symfony (v2+, v3+)
+
+Look at your configuration directory, and edit `config_dev.yml` or `config_prod.yml` depending on your needs: 
+
+```yaml
+ # app/config/config.yml
+monolog:
+
+# Uncomment this section, if you want to use a processor
+#       processor: 
+#           session_processor:
+#               class: Acme\Bundle\MonologBundle\Log\SessionRequestProcessor
+#            arguments:  [ @session ]
+#            tags:
+#               - { name: monolog.processor, method: processRecord }
+          
+          
+    json_formatter:
+        class: Monolog\Formatter\JsonFormatter
+          
+    handlers:
+    
+            # Log shipper configuration
+        to_json_files:
+            # log to var/logs/(environment).log
+            type: stream
+            path: "%kernel.logs_dir%/%kernel.environment%.log"
+            # inclues all channels (doctrine, errors, etc.)
+            channels: ~
+            # use json formatter
+            formatter: monolog.json_formatter
+            # logs alls events (debug trought fatal)
+            level: debug
+```
+
+#### PPI
+
+Look at your configuration directory, and edit `config_dev.yml` or `config_prod.yml` depending on your needs: 
+
+```yaml
+monolog:
+    handlers:
+    
+        # Log shipper configuration
+        to_json_files:
+            # log to var/logs/(environment).log
+            type: stream
+            path: "%kernel.logs_dir%/%kernel.environment%.log"
+            # use json formatter
+            formatter: monolog.json_formatter
+            # logs alls events (debug trought fatal)
+            level: debug
+```
+
+#### Laravel
+
+```php
+ <?php 
+
+//file: bootstrap/app.php
+$app->configureMonologUsing(function($monolog) {
+    $monolog->pushHandler(...);
+    
+  // configure your logger as below 
+});
+
+return $app;
+```
+
+#### Silex
+
+```php
+<?php
+
+// file: bootstrap
+$app->extend('monolog', function($monolog, $app) {
+    $monolog->pushHandler(...);
+    // configure your logger as below 
+    return $monolog;
+});
+```
+
+#### Lumen
+
+```php
+<?php 
+
+//file: bootstrap/app.php
+$app->configureMonologUsing(function($monolog) {
+    $monolog->pushHandler(...);
+    // configure your logger as below 
+});
+
+return $app;
+```
+
+#### CakePHP
+
+First, add this dependency to the `composer.json` file and
+run `composer update`.
+
+```json
+{
+    "require": {
+        "cakephp/monolog": "*"
+    }
+}
+```
+
+Then, start by creating a logging configuration  file (i.e. `app/Config/log.php`) that you include early in your `app/Config/bootstrap.php`:
```

Comment:
Two comments here:

1) There's an extra space between `configuration  file`
2) I'd remove the word `early`

> Then, start by creating a logging configuration file (i.e. `app/Config/log.php`) that you include in your `app/Config/bootstrap.php`:

Reply:
👌